docs/sphinx/examples/python/building_kernels.py

```diff
@@ -0,0 +1,266 @@
+# # Building Kernels
+#
+# This section will cover the most basic CUDA-Q construct, a quantum kernel. Topics include, building kernels, initializing states, and applying gate operations.
```

Comment:
This text should move out of the code and into the page text, as it is common for both Python and C++.